src/main/java/DayOne.java

```diff
@@ -5,9 +5,11 @@
 public class DayOne {
     Double totalFuel =0.0;
     public Double getFuel(String fuels) {
+        //could just use array
         List<String> listOfFuels = Arrays.asList(fuels.split("\n"));
         for (String fuel:
              listOfFuels) {
+            //redundant floor valueOf
```

Comment:
This inline call to `replace` is a bit messy, and makes it harder to read, can we do it another way.  Also because of the way integer arithmetic works, I think the call to `Math.floor` is redundant.  Finally there's an alternate method to `Integer.valueOf` called `Integer.parseInt` that just returns a primitive `int` which I think is more appropriate in this circumstance.